### PR TITLE
chore: use vuepress-shared and update ts resolution

### PR DESCRIPTION
**Pull request by Mister-Hope**

### CODE REVIEW

**Review by DreamOfIce — COMMENTED** [review]
Why do you move this to the bottom?

File: src/node/utils.ts [hunk 1]
```diff
@@ -76,30 +75,12 @@ const insertAfterFrontmatter = (content: string, data: string) => {
...
-  );
+const logger = new Logger(PLUGIN_NAME);
 
 export {
   PLUGIN_NAME,
   addComponent,
   getLocales,
-  insertAfterFrontmatter,
   logger,
+  insertAfterFrontmatter,
```
Comment (DreamOfIce) [inline src/node/utils.ts:85]: Why do you move this to the bottom?

**Comment by Mister-Hope** [comment]
> Why do you move this to the bottom?

What do you mean by 'this' and 'bottom'

**Comment by DreamOfIce** [comment]
> > Why do you move this to the bottom?
> 
> What do you mean by 'this' and 'bottom'

Sorry, I posted it in the wrong place, please see the review

File: src/node/utils.ts (hunk 1 above)
Comment (Mister-Hope) [inline src/node/utils.ts:85]:
```suggestion
  insertAfterFrontmatter,
  logger,
```

**Pull request merged**